libs/platform/src/lib/components/menu/menu-item.component.ts

```diff
@@ -31,6 +31,8 @@ export class MenuItemComponent implements OnDestroy, FocusableOption {
 
     // Add Fundamental-Styles class for menu item
```

Comment:
These shouldn't be normal comments but typedocs

Reply:
Added typedocs.